stable/syncthing/SCALE/questions.yaml

```diff
@@ -141,17 +141,120 @@ questions:
                       schema:
                         type: int
                         default: 8384
-                        editable: false
+                        editable: true
+                        hidden: false
+                    - variable: nodePort
+                      label: "Host nodePort to expose to (optional)"
+                      description: "Only gets used when nodePort is selected"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 36024
+                        required: true
+        - variable: listeners
+          label: "Syncthing listening ports"
+          description: "This service is used to process incoming connections directly to this Syncthing instance"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable the service"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: false
+              - variable: type
+                label: "Service type"
+                description: "ClusterIP's are only internally available, nodePorts expose the container to the host node System"
+                schema:
+                  type: string
+                  default: "NodePort"
+                  enum:
+                    - value: "NodePort"
+                      description: "NodePort"
+                    - value: "ClusterIP"
+                      description: "ClusterIP"
+              - variable: port
+                label: "TCP port"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: protocol
+                      label: "Port Type"
+                      schema:
+                        type: string
+                        default: "UDP"
+                        hidden: true
+                        enum:
+                          - value: TCP
+                            description: "TCP"
+                          - value: "UDP"
```

Comment:
Copied from transmission, but why is "UDP" in quotes and TCP isn't? Is this necessary?

Reply:
Not everything is perfect. 
Nor should you blindly copy and paste for that reason ;-)